src/EditorFeatures/TestUtilities/EditAndContinue/EditAndContinueTestHelpers.cs

```diff
@@ -265,8 +275,8 @@ public static void VerifyDiagnostics(IEnumerable<RudeEditDiagnosticDescription>
         {
             // string comparison to simplify understanding why a test failed:
             AssertEx.Equal(
-                expectedSemanticEdits.Select(e => $"{e.Kind}: {e.SymbolProvider(newCompilation)}"),
-                actualSemanticEdits.NullToEmpty().Select(e => $"{e.Kind}: {e.Symbol.Resolve(newCompilation).Symbol}"),
+                expectedSemanticEdits.Select(e => $"{e.Kind}: {e.NewSymbolProvider(newCompilation)}"),
```

Comment:
>NewSymbolProvider [](http://example.com/codeflow?start=65&length=17)

Doesn't seem right. Shouldn't the current representation of the symbol stay and a new added to the string that represents the container type of a delete edit?